ai/smart-labeling/how-to/create-labeling-workspace.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: How to create a labeling workspace
+description: This page explains how to create a labeling workspace
+dates:
+  validation: 2021-11-05
+  posted: 2021-11-05
+---
+
+<Message type="requirement">
+
+- You have an account and are logged into the [Scaleway Console](https://console.scaleway.com)  
```

Comment:
Another requirement is that they have created an object storage bucket. Maybe we could also add a few lines of introduction, explaining the principle of creating an object storage bucket and uploading the images and videos to be labelled (with a link to the relevant how tos for creating buckets / uploading files) , and then creating a labeling workspace and associating the bucket with it. I believe that it is imperative that hte bucket be created BEFORE the workspace, else they have to delete the workspace and start all over again.